MAINTAINERS.md

```diff
@@ -133,23 +172,34 @@ All new issues will need to be triaged, and pull requests must be examined.  Mai
 
 > If you see an issue or PR that could use some labels, please add them!
 
-The following list is incomplete, but it's better than nothing:
-
 ### Semantic Versioning
 
-*All* issues which will be resolved by commit(s) should have one of these three labels:
+The TL;DR of Semantic Versioning is:
+
+- MAJOR version when you make incompatible API changes,
+- MINOR version when you add functionality in a backwards-compatible manner, and
+- PATCH version when you make backwards-compatible bug fixes.
+
+Pull requests *must* have one of these three (3) labels:
+
+- `semver-patch` for backwards-compatible bug fixes, documentation, or anything which does not affect a "production" (`npm install mocha`) installation of Mocha
+- `semver-minor` for backwards-compatible new features or usability/interface enhancements
+- `semver-major` for backwards-incompatible ("breaking") changes to the API
 
-- `semver-patch` for bug fixes, documentation, development environment changes, CI and tests
-- `semver-minor` for enhancements ("features")
-- `semver-major` for backwards-incompatible ("breaking") changes to the *output*, *API*, or environment support
+A PR which introduces a breaking change is considered to be `semver-major`, *regardless* of whether it's a bug fix, feature, or whatever.
 
-To be clear:
+For the purposes of the above definitions, Mocha has some unique considerations, and includes the following in its definition of "API":
 
-- Features, bugs, and updated tests are examples of issues which will be resolved by commit(s).
-- Support questions, unconfirmed bugs and [bikeshedding](https://en.wikipedia.org/wiki/Law_of_triviality) are examples of issues which do *not* need `semver-*` labels, since they won't necessarily result in any changes to the codebase.
-- Pull requests *do not* need `semver-*` labels, *unless there is no associated issue* (PRO TIP: make an issue!)
-- An issue or PR which will introduce a breaking change will be `semver-major`, *regardless* of any other label.
-- **Breaking changes to private APIs will be `semver-major`, if and only if they are known to be consumed by actively developed project(s).**
+1. Mocha's *documented*, programmatic interface which *is not explicitly marked as "private"*
```

Comment:
@plroebuck `@api` was in there in the first place because that's what TJ put in there.  I don't know why; I don't know of anything that consumes it.  As far as I can tell, the *intent* of e.g., `@api private` was the same as what `@private` gives you.  Maybe he thought `@private` was confusing (it is--it's trying to represent two separate concepts).

I'm not sure I understand what you'd like to see here.  "...explicitly tagged with `@private`"?